website/static/js/comment.js

```diff
@@ -40,6 +77,9 @@ var relativeDate = function(datetime) {
 };
 
 var notEmpty = function(value) {
+    if (value === '<br>')  {
```

Comment:
Should you also handle a `<br>` with any surrounding whitespace, i.e. `'<br> '`?